library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -178,6 +178,15 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         return MessageV3(client, message)
     }
 
+    fun updateConsentState(state: ConsentState) {
+        val consentState = ConsentState.toFfiConsentState(state)
+        libXMTPGroup.updateConsentState(consentState)
```

Comment:
While we still dont have history we should probably update the V2 one here as well 🤔